be/src/storage/lake/starlet_group_assigner.cpp

```diff
@@ -27,9 +31,18 @@ Status StarletGroupAssigner::list_group(std::vector<std::string>* groups) {
     }
 
     std::vector<staros::starlet::ShardInfo> shards = g_worker->shards();
-    auto group_id = 1001; // TODO get group id from shardinfo
-    for (const auto& shard : shards) {
-        std::string starlet_group = fmt::format("{}{}/{}", kStarletPrefix, shard.obj_store_info.uri, group_id);
+    for (auto& shard : shards) {
```

Comment:
const auto

Reply:
According to my understanding, const auto will cause a copy of shardinfo.